test/Tester/StreamingTests/EHImplicitSubscriptionStreamRecoveryTests.cs

```diff
@@ -88,14 +46,14 @@ public EHImplicitSubscriptionStreamRecoveryTests()
         public async Task Recoverable100EventStreamsWithTransientErrorsTest()
         {
             logger.Info("************************ EHRecoverable100EventStreamsWithTransientErrorsTest *********************************");
-            await runner.Recoverable100EventStreamsWithTransientErrors(GenerateEvents, ImplicitSubscription_TransientError_RecoverableStream_CollectorGrain.StreamNamespace, 4, 100);
+            await CheckpointSettings.AwaitAndCleanup(runner.Recoverable100EventStreamsWithTransientErrors(GenerateEvents, ImplicitSubscription_TransientError_RecoverableStream_CollectorGrain.StreamNamespace, 4, 100));
```

Comment:
Thanks for catching this.  This was a brain dead workaround I through together mid testing because I didn't know how to tell xunit to cleanup after a test run.  I didn't intend to leave it this way.  I'll learn xunit :)